### PR TITLE
[sncf] Align XML description and default translations

### PR DESCRIPTION
**Pull request by lolodomo**
Re-run of the i18n tool will then trigger no change.

Signed-off-by: Laurent Garnier <lg.hc@free.fr>

### CODE REVIEW

**Review by clinique — APPROVED** [review]

**Review by clinique — APPROVED** [review]

File: bundles/org.openhab.binding.sncf/src/main/resources/OH-INF/thing/station.xml [hunk 1]
```diff
@@ -10,21 +10,25 @@
 		</supported-bridge-type-refs>
 
 		<label>Station</label>
+		<description>Represents a station hosting some transportation mode.</description>
 
 		<channel-groups>
 			<channel-group id="arrivals" typeId="passage">
 				<label>Next Arrival</label>
+				<description>Informations regarding next arrival at the station.</description>
```
Comment (wborn) [inline bundles/org.openhab.binding.sncf/src/main/resources/OH-INF/thing/station.xml:18]:
```suggestion
				<description>Information regarding the next arrival at the station.</description>
```

File: bundles/org.openhab.binding.sncf/src/main/resources/OH-INF/thing/station.xml [hunk 2]
```diff
@@ -10,21 +10,25 @@
...
 			</channel-group>
 			<channel-group id="departures" typeId="passage">
 				<label>Next Departure</label>
+				<description>Informations regarding next departure from the station.</description>
```
Comment (wborn) [inline bundles/org.openhab.binding.sncf/src/main/resources/OH-INF/thing/station.xml:22]:
```suggestion
				<description>Information regarding the next departure from the station.</description>
```

**Review by wborn — CHANGES_REQUESTED** [review]

File: bundles/org.openhab.binding.sncf/src/main/resources/OH-INF/thing/station.xml (hunk 1 above)
Comment (lolodomo) [reply bundles/org.openhab.binding.sncf/src/main/resources/OH-INF/thing/station.xml:18]: Fixed

File: bundles/org.openhab.binding.sncf/src/main/resources/OH-INF/thing/station.xml (hunk 2 above)
Comment (lolodomo) [reply bundles/org.openhab.binding.sncf/src/main/resources/OH-INF/thing/station.xml:22]: Fixed

**Review by wborn — APPROVED** [review]
Thank you!

**Pull request merged**